app/api/admin_sales/events.py

```diff
@@ -0,0 +1,60 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema
+from flask_rest_jsonapi import ResourceList
+
+from app.api.bootstrap import api
+from app.models import db
+from app.models.event import Event
+from app.models.order import Order, OrderTicket
+
+from app.api.admin_sales.utils import summary
+
+
+class AdminSalesByEventsSchema(Schema):
```

Comment:
Isn't this already extremely similar to https://github.com/fossasia/open-event-server/blob/6b1f5db3ed3c8cf1781dd09830bcb6ba60135205/app/api/admin_statistics_api/events.py#L15 ?

Reply:
`AdminSalesByEventsSchema `
`AdminStatisticsEventSchema` 
class names differs. @Kreijstal

Reply:
And they do totally different things